storage.go

```diff
@@ -37,6 +38,9 @@ func (s *Storage) completeMultipart(ctx context.Context, o *Object, parts []*Par
 	if err != nil {
 		return
 	}
+
+	o.Mode &= ^ModePart
+	o.Mode |= ModeRead
```

Comment:
Maybe we can extract to function `Add` && `Del`?

```
o.Mode.Add(ModeRead)
o.Mode.Del(ModePoart)
```

Reply:
Should I add function `Add` and `Del` in `go-storage` or just add in utils.go?

Reply:
Let's add them in go-storage.